armi/bookkeeping/db/database3.py

```diff
@@ -1776,6 +2035,65 @@ def writeToDB(self, h5group):
             runLog.error("Failed to create datasets in: {}".format(h5group))
             raise
 
+    @staticmethod
+    def computeAncestors(serialNum, numChildren, depth=1) -> List[Optional[int]]:
+        """
+        Return a list containing the serial number of the parent corresponding to each
+        object at the given depth.
```

Comment:
can you explain that slightly more? I see a serialNum input that is hard to reconcile with this description quickly

Reply:
I at least added argument doc strings and more info on `depth`.